app/views/catalog/_show_header.html.erb

```diff
@@ -0,0 +1,24 @@
+<% # Override of BlacklLight 7.4.1 -%>
+<% # bookmark/folder functions -%>
+<%= render_document_heading(document, :tag => :h1) %>
+<span class="availability-badge"></span>
+
+<script type="text/javascript">
+  $.ajax({
+    url: "/alma_availability/<%= document.id %>.json",
+    type: "get",
+    dataType: 'json',
+    success: function(data) { 
+      var availabilityHtml = '';
+      var physicalExists = data['<%= document.id %>']['physical']['exists'];
+      var physicalAvailable = data['<%= document.id %>']['physical']['available'];  
+
+      if (physicalExists == true && physicalAvailable == true) {
+        availabilityHtml = '<span class="badge badge-success">Available</span>';
+      } else if (physicalExists == true && physicalAvailable == false) {
+        availabilityHtml = '<span class="badge badge-danger">UnAvailable</span>';
```

Comment:
Is there a reason why "A" is capitalized in unavailable?

Reply:
I have no clue why I did that. Some weird artistic flourish of mine? Anyways, I'll fix it.

Reply:
A new commit has been pushed with the corrections.